src/foam/core/Validation.js

```diff
@@ -354,30 +354,19 @@ foam.CLASS({
       name: 'validationPredicates',
       factory: function() {
         var self = this;
-        var ret = [
+        if ( ! this.required ) return [];
+        return [
           {
             args: [this.name],
             predicateFactory: function(e) {
                 return e.REG_EXP(
                   self,
                   /^(?:\+?1[-.●]?)?\(?([0-9]{3})\)?[-.●]?([0-9]{3})[-.●]?([0-9]{4})$/
                  );
-              },
-              errorString: 'Please enter a valid phone number'
-            }
+            },
+            errorString: 'Please enter a valid phone number'
```

Comment:
Better to use a Message so we can internationalize in the future. Not going to block PR, just saying.